packages/core/content-manager/admin/src/preview/components/PreviewHeader.tsx

```diff
@@ -183,6 +184,9 @@ const UnstablePreviewHeader = () => {
     documentId: document.documentId,
     document,
     meta,
+    onPreview: () => {
+      iframeRef?.current?.contentWindow?.postMessage({ type: 'strapiUpdate' });
+    },
   } satisfies DocumentActionProps;
```

Comment:
Could you rename this object something like documentActionProps? Otherwise just reading props I'd think it's the component's props (UnstablePreviewHeader)